### PR TITLE
Update client4.ts: fix typo 

### PR DESCRIPTION
**Pull request by wiggin77**
Fix typo. Not sure how this affects analytics reporting, but I suspect no issue.

### CODE REVIEW

File: src/client/client4.ts [hunk 1]
```diff
@@ -1391,7 +1391,7 @@ export default class Client4 {
     };
 
     updateChannelNotifyProps = async (props: any) => {
-        this.trackEvent('api', 'api_users_update_channel_notifcations', {channel_id: props.channel_id});
+        this.trackEvent('api', 'api_users_update_channel_notifications', {channel_id: props.channel_id});
```
Comment (saturninoabril) [inline src/client/client4.ts:1394]: @jasonblais I'm adding you as a reviewer as I believe you have more context on analytics/tracking events.  TIA.

Comment (enahum) [reply src/client/client4.ts:1394]: if this is valid, please do file a ticket for mobile

**Comment by jasonblais** [comment]
@wiggin77 This is ready to merge, but I'm not authorized to, FYI

**Pull request merged**